jquery-ajax-localstorage-cache.js

```diff
@@ -7,6 +7,13 @@
      * or one generated from the url, the type and, if present, the data.
      */
     var genCacheKey = function (options) {
+        //in case cachey key is required to be generated dynamically by a function
```

Comment:
All variable declarations should be at the top of the function, due to var hoisting.